infrastructure-formula/infrastructure/libvirt/domains.sls

```diff
@@ -26,6 +26,7 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 {%- else -%}
   {%- do salt.log.debug('libvirt.domains: running non-orchestrated') -%}
   {%- set domain = grains['domain'] -%}
+  {%- set do_all_domains = salt['pillar.get']('infrastructure:libvirt:domains:do_all', false) -%}
```

Comment:
Should `do_all` be mentioned / documented in pillar.example.yml?

Besides this, the changes look good.